erpnext/crm/onboarding_step/newsletter/newsletter.json

```diff
@@ -0,0 +1,21 @@
+{
+ "action": "Go to Page",
+ "action_label": "Click here to learn more about Newsletter",
+ "creation": "2021-07-21 00:28:41.649039",
+ "description": "# Newsletter\nA newsletter is an email that is sent to a specific group of subscribers periodically.\n\nYou can draft Newsletters in ERPNext, with text and images, tag the Email Group to whom it will be sent and schedule to send it at a preferred time.\n",
```

Comment:
Doesn't navigate to the newsletter. Check comment for Email Group and apply the same.